core/src/main/java/org/elasticsearch/common/xcontent/XContentBuilder.java

```diff
@@ -944,6 +944,10 @@ public XContentBuilder dateValueField(XContentBuilderString rawFieldName, XConte
         return this;
     }
 
+    public XContentBuilder timeValueField(String rawFieldName, String readableFieldName, TimeValue timeValue) throws IOException {
```

Comment:
Yeah, totally outside the scope of this issue, just something I notice. Documenting sounds good to me.